### ISSUE TITLE
Option to disable gap fill

### ISSUE THREAD

**Issue by NVNDO** [bambulab/BambuStudio#4090]
**Problem**
Gab infill creates unwanted scars and/or creates zits. Many of the gaps detected by the slicer are already filled by the top-layer, so "gap infill" is many times not only unnecessary, but also wastes time and destroys the top layer.

**Solution**
Being able to disable / deactivate "gap infill"

**Alternatives**
None. In order to get clean top layers on smaller objects we need this feature. 

**Additional context**
"Avoid crossing walls" seems to be ignored as when filling gaps the nozzle crosses wherever it wants.  

![Screenshot (613)](https://github.com/bambulab/BambuStudio/assets/73851760/a8b6b5fd-976a-479e-b5dc-5059c0228cac)
![thumbnail_IMG_6005](https://github.com/bambulab/BambuStudio/assets/73851760/1f80b445-fe66-41b7-a6b9-5e7074140143)

**Comment by frasier2k** [comment]
I also would like to have the threshold field for the gap fill like Orca Slicer had it.

**Comment by joojoooo** [comment]
Yeah, a much needed feature, the only solution I found is switching to OrcaSlicer to use "filter out tiny gaps"

<img width="695" height="415" alt="Image" src="https://github.com/user-attachments/assets/1c87051a-e935-47a6-b071-5dfdd275e144" />

I tried using  arachne to fix it but it's even worse on another point, places too many seams:
<img width="1323" height="832" alt="Image" src="https://github.com/user-attachments/assets/914fa4ae-9834-4b0e-8817-7831a54bf26f" />

While classic doesn't place the seams there:
<img width="913" height="658" alt="Image" src="https://github.com/user-attachments/assets/b15784f9-5970-48a0-8f05-341239806897" />

So the only solution is classic without small gap infill, which bambustudio can't do :(

**Comment by Highpockets003** [comment]
I've updated to 2.3.0. Where exactly is the download file for 2.3.1 beta? Can't find that page or download.

Also, slicing my model with "Apply gap infill" set to "nowhere" still yields gap infill. :(

![Image](https://github.com/user-attachments/assets/ba6fa4a0-7b8e-43eb-8fb1-b199360a86a3)
